src/main/java/com/iridium/iridiumteams/commands/BlockValueCommand.java

```diff
@@ -0,0 +1,107 @@
+
+    @Override
+    public boolean execute(U user, String[] args, IridiumTeams<T, U> iridiumTeams) {
+
+        Player player = user.getPlayer();
+        Optional<T> team;
+
+        BlockValuesTypeSelectorInventoryConfig blockValuesTypeSelectorInventoryConfig = iridiumTeams.getInventories().blockValuesTypeSelectorGUI;
+
+        String teamArg;
+        if (args.length > 1) {
+            teamArg = args[1];
+        } else {
+            teamArg = player.getName();
+        }
+
+        team = iridiumTeams.getTeamManager().getTeamViaNameOrPlayer(teamArg);
+
+        if (!team.isPresent() && args.length > 1) {
+            player.sendMessage(StringUtils.color(iridiumTeams.getMessages().teamDoesntExistByName.replace("%prefix%", iridiumTeams.getConfiguration().prefix)));
+            return false;
+        }
+
+        if (!team.isPresent()) {
+            player.sendMessage(StringUtils.color(iridiumTeams.getMessages().dontHaveTeam.replace("%prefix%", iridiumTeams.getConfiguration().prefix)));
+            return false;
+        }
+
+        TeamSetting teamSetting = iridiumTeams.getTeamManager().getTeamSetting(team.get(), SettingType.VALUE_VISIBILITY.getSettingKey());
+
+        if (teamSetting == null) {
+            player.sendMessage(StringUtils.color(iridiumTeams.getMessages().invalidPermission.replace("%prefix%", iridiumTeams.getConfiguration().prefix)));
+            return false;
+        }
+
+        if (teamSetting.getValue().equalsIgnoreCase("Private") && !iridiumTeams.getTeamManager().getTeamMembers(team.get()).contains(user) && !user.isBypassing()) {
+            player.sendMessage(StringUtils.color(iridiumTeams.getMessages().teamIsPrivate.replace("%prefix%", iridiumTeams.getConfiguration().prefix)));
+            return false;
+        }
+
+        if (args.length == 0) {
+            player.openInventory(new BlockValuesTypeSelectorGUI<>(teamArg, player.getOpenInventory().getTopInventory(), iridiumTeams).getInventory());
+            return true;
+        }
+
+        switch (args[0]) {
+            case ("blocks"): {
+                if (blockValuesTypeSelectorInventoryConfig.blocks.enabled) {
+                    player.openInventory(new BlockValueGUI<>(team.get(), BlockValueGUI.BlockValueType.BLOCK, player.getOpenInventory().getTopInventory(), iridiumTeams).getInventory());
+                    return true;
+                }
+            }
+            case ("spawners"): {
+                if (blockValuesTypeSelectorInventoryConfig.spawners.enabled) {
+                    player.openInventory(new BlockValueGUI<>(team.get(), BlockValueGUI.BlockValueType.SPAWNER, player.getOpenInventory().getTopInventory(), iridiumTeams).getInventory());
+                    return true;
+                }
+            }
+            default: {
+                player.openInventory(new BlockValuesTypeSelectorGUI<>(teamArg, player.getOpenInventory().getTopInventory(), iridiumTeams).getInventory());
+                return true;
+            }
```

Comment:
I think by default we should show the correct syntax instead of opening the select gui

Reply:
normally i would agree wth you, but for blockvalues i think it makes sense to open the gui automatically since we havent chosen a page to look at, same with missions? how do we do it there lol